tests/classes/search/ArticleSearchTest.php

```diff
@@ -250,6 +250,7 @@ private function registerMockArticleSearchDAO()
     private function registerMockIssueDAO($published = true)
     {
         // Mock an IssueDAO.
+        // TODO: Will this work with Repo/EntityDAO and/or are these tests still used (as they're skipped)?
```

Comment:
Two questions here:

1. Does this need to be updated as all the tests here are skipped?
2. What's the best approach for mocking this as the functionality would require the Issue Repository rather than just the DAO.

Reply:
cc @asmecher

Reply:
In general I think it should be possible to set up testbeds for database calls by mocking the DAO alone; it's the only place where queries are actually executed.

In this specific case, the `main` branch's `registerMockIssueDAO` just mocks `IssueDAO::getById`; the new equivalent would be `DAO::get` (and `Repository` calls would be passed through to this). I think this can just be adapted as is without much work being required.